fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -1622,6 +1622,11 @@ def id_to_token(self, id: int) -> Optional[str]:
         Returns:
             :obj:`Optional[str]`: An optional token, :obj:`None` if out of vocabulary
         """
+        if not isinstance(id, int):
```

Comment:
I find it useful since `self.decoder_dict.get(id, None)` (next line below) doesn't enforce a type, but will just return `None` (I fell on it..)

Reply:
Double-check with Moshiko. I think we use non-integer numbers (they're still N, but of a different type) for IDs in parts of our code.

Reply:
I think it is int.
@SagiPolaczek, did you test it in action?

Reply:
@mosheraboh 
Yes, it works with my discrete diffusion training run.